### ISSUE TITLE
This is a pool question more than issue.

### ISSUE THREAD

**Issue by lesjokolat** [MPOS/php-mpos#2782]
My mining software can take a new job from pool, I then am able to process the job and find multiple valid solutions. The issue is will only 1 solution be considered valid(the first) all others get considered invalid, can I send all possible valid solutions in any way so as not to error? Does the pool need to be reconfigured to allow this behavior on jobs submitted back?

thanks

**Comment by r4sas** [comment]
Not MPOS issue. Ask your stratum software developer.

**Comment by lesjokolat** [comment]
Well I am working with the developer. So let me rephrase then:

We are getting invalid jobs back when they are actually valid solutions.

Does each job from pool software only allow for one valid solution? Does the pool software allow one to submit and have accepted more than just one possible solution without errors?

thanks

**Comment by r4sas** [comment]
Again: not a MPOS issue.
MPOS is only working with data in database, which stored by your stratum software. If MPOS see that share marked as invalid in database, it will print it and process as invalid.
Please read project wiki.
